examples/todo-list/src/repositories/todo.repository.ts

```diff
@@ -36,55 +35,20 @@ export class TodoRepository extends DefaultCrudRepository<
       'todoList',
       todoListRepositoryGetter,
     );
-  }
-
-  async find(
```

Comment:
I don't see any mocha tests have changed. Some stub repo test uses `find`. 

https://github.com/strongloop/loopback-next/blob/master/examples/todo-list/src/__tests__/unit/controllers/todo-list.controller.unit.ts#L58

Should it still use `find` now that a repo no longer has this function?

Reply:
So the `find` method here just overrode [the one in `DefaultCrudRepository`](https://github.com/strongloop/loopback-next/blob/resolver-docs/packages/repository/src/repositories/legacy-juggler-bridge.ts#L359-L369) and now that it's removed, it'll use that one again.